pkgs/tools/networking/rcon/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, glib, libbsd, check, pcre }:
+
+stdenv.mkDerivation rec {
+  pname = "rcon";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "n0la";
+    repo = "rcon";
+    rev = version;
+    sha256 = "1jsnmsm2qkiv8dan1yncx0qp6zfkcbyvf81c7xwpv6r499ijw1nb";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
```

for new packages, we prefer not to use the alias
```suggestion
  nativeBuildInputs = [ cmake pkg-config ];
```